pandas/tests/arrays/sparse/test_arithmetics.py

```diff
@@ -26,6 +26,8 @@ def mix(request):
     return request.param
 
 
+# FIXME: There are not SparseArray tests. There are numpy array tests.
```

Comment:
IIUC these are SparseArray tests just not useful ones bc they don't have non-trivial indexes?  if correct, can you clarify this for future readers